src/Apps/Order/Routes/Offer/index.tsx

```diff
@@ -259,16 +258,8 @@ export class OfferRoute extends Component<OfferProps, OfferState> {
   }
 }
 
-const OfferRouteWrapper = props => (
-  <ContextConsumer>
-    {({ mediator }) => {
-      return <OfferRoute {...props} mediator={mediator} />
-    }}
-  </ContextConsumer>
-)
-
 export const OfferFragmentContainer = createFragmentContainer(
-  injectDialog(OfferRouteWrapper),
+  injectDialog(trackPageViewWrapper(OfferRoute)),
```

Comment:
Speaking of which, `injectDialog` can also go up there:

```js
@track() 
@trackPageView()
@injectDialog()
export class OfferRoute extends React.Component { ... }
```

Reply:
Note that the `trackPageView` and `injectDialog` decorators aren't being called (`()`) since they're not designed to accept configurtation. If they were, its just another level of currying:

```js
export function trackPageView(config) {
  return (ComponentToWrap) => {
     export class TrackPageView extends React.Component { ... }
  }
}
```